src/shared/firewall-util-nft.c

```diff
@@ -34,7 +34,26 @@
 
 #define UDP_DPORT_OFFSET 2
 
-static sd_netlink_message **netlink_message_unref_many(sd_netlink_message **m) {
+/* for test-firewall-util */
+static const char* nft_table_name(void) {
+        static const char *cached = NULL;
```

Comment:
shouldn't the static be a bool? if the env var is not set, it will always be checked, as it returns NULL, no?

Reply:
If unset, then `NFT_SYSTEMD_TABLE_NAME` will be assigned to `cached`. Hence, `if (!cached)` will not pass on later calls.

Reply:
duh, I missed the ternary, nvm